homeassistant/components/knx/expose.py

```diff
@@ -116,12 +114,14 @@ def shutdown(self) -> None:
     def _get_expose_value(self, state: State | None) -> StateType:
         """Extract value from state."""
         if state is None or state.state in (STATE_UNKNOWN, STATE_UNAVAILABLE):
-            return None
+            return self.expose_default
         value = (
             state.state
             if self.expose_attribute is None
             else state.attributes.get(self.expose_attribute)
         )
+        if value is None:
+            value = self.expose_default
```

Comment:
```suggestion
            return self.expose_default
        value = (
            state.state
            if self.expose_attribute is None
            else state.attributes.get(self.expose_attribute, self.expose_default)
        )
```
A default for the attributes should suffice. `state.state` can not be `None` anyway.